docs/en/Tiered-Locality.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: global
+title: Tiered Locality
+nickname: Tiered Locality
+group: Features
+priority: 1
+---
+
+* Table of Contents
+{:toc}
+
+# Tiered Identity
+
+In Alluxio, each entity (masters, workers, clients) has a *Tiered Identity*. A tiered
+identity is an address tuple such as `(node=..., rack=...)`. The elements
+of the tuple are ordered from most specific to least specific, so if two entities have
+the same node name, they are expected to also have the same rack name. Alluxio uses tiered
+identities to optimize for locality. For example, when the client wants to read a
+file from the UFS, it will prefer to read through an Alluxio worker on the same node. 
+If there is no local worker in the first tier (node), the next tier (rack) will be checked 
+to prefer rack-local data transfer. If no workers share a node or rack with the client, an
+arbitrary worker will be chosen.
+
+# Configuration
+
+If the user does nothing to provide tiered identity info, each entity will
+perform a localhost lookup to set its node-level identity info. To explicitly configure
+the value for a locality tier, set the configuration property
```

Comment:
In general, what happens if a tier is not specified (either by config or script)?

Reply:
it will be as though the tier doesn't exist, or as though the value is different from the value for all other entities. I've updated the doc

Reply:
I understand an unset tier will not match with another set tier, but will 2 unset tiers match with each other?

Reply:
2 unset tiers will not match with each other